docker/docker-compose-examples/analytics/docker-compose.yml

```diff
@@ -122,14 +122,16 @@ services:
       - CUBEJS_DB_NAME=${CH_DB:-clickhouse_test_db}
       - CUBEJS_DB_USER=${CH_USER:-clickhouse_test_user}
       - CUBEJS_DB_PASS=${CH_PWD:-clickhouse_password}
-      - CUBEJS_JWK_URL=${JWKS_URL:-http://keycloak:61111/realms/dotcms/protocol/openid-connect/certs}
+      - CUBEJS_JWK_URL=${JWKS_URL:-http://host.docker.internal:61111/realms/dotcms/protocol/openid-connect/certs}
       - CUBEJS_JWT_AUDIENCE=api-dotcms-analytics-audience
-      - CUBEJS_JWT_ISSUER=${AUTH_SERVER_URL:-http://keycloak:61111/realms/dotcms}
+      - CUBEJS_JWT_ISSUER=${AUTH_SERVER_URL:-http://localhost:61111/realms/dotcms}
```

Comment:
I think this is not OK

Reply:
Here is the JWT 

<img width="549" alt="image" src="https://github.com/user-attachments/assets/d854c988-a5fb-472b-bcaf-639f811a9976" />

ehat this property does is compare the value of the iss attribute with the variable, it not really a URL to hit